src/Controller/JobAdvertsController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace eLife\Journal\Controller;
+
+use DateTimeImmutable;
+use eLife\ApiSdk\Collection\Sequence;
+use eLife\ApiSdk\Model\JobAdvert;
+use eLife\Journal\Helper\Callback;
+use eLife\Journal\Helper\Paginator;
+use eLife\Journal\Pagerfanta\SequenceAdapter;
+use eLife\Journal\ViewModel\Paragraph;
+use eLife\Patterns\ViewModel\ContentHeader;
+use eLife\Patterns\ViewModel\ListingTeasers;
+use eLife\Patterns\ViewModel\Teaser;
+use Pagerfanta\Pagerfanta;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use function GuzzleHttp\Promise\promise_for;
+
+final class JobAdvertsController extends Controller
+{
+    public function listAction(Request $request) : Response
+    {
+        $page = (int) $request->query->get('page', 1);
+        $perPage = 10;
+
+        $arguments = $this->defaultPageArguments($request);
+
+        $latest = promise_for($this->get('elife.api_sdk.job_adverts')
+            ->show('open'))
+            ->then(function (Sequence $sequence) use ($page, $perPage) {
+                $pagerfanta = new Pagerfanta(new SequenceAdapter($sequence, $this->willConvertTo(Teaser::class)));
+                $pagerfanta->setMaxPerPage($perPage)->setCurrentPage($page);
+
+                return $pagerfanta;
+            });
+
+        $arguments['title'] = 'eLife Jobs';
+
+        $arguments['paginator'] = $latest
+            ->then(function (Pagerfanta $pagerfanta) use ($request) {
+                return new Paginator(
+                    'Browse our open job adverts',
+                    $pagerfanta,
+                    function (int $page = null) use ($request) {
+                        $routeParams = $request->attributes->get('_route_params');
+                        $routeParams['page'] = $page;
+
+                        return $this->get('router')->generate('job-adverts', $routeParams);
+                    }
+                );
+            });
+
+        $arguments['listing'] = $arguments['paginator']
+            ->then($this->willConvertTo(ListingTeasers::class, ['heading' => 'Latest', 'type' => 'job-adverts', 'emptyText' => 'No vacancies at present.  Feel free to send your CV to hr@elifesciences.org']));
```

Comment:
Do we have a policy of not linking email addresses now? (Notice we're not to do it on profiles.)